source/common/http/conn_manager_impl.h

```diff
@@ -164,6 +164,9 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     // current filter instead of the next one. If true, filter iteration starts with the current
     // filter. Otherwise, starts with the next filter in the chain.
     bool iterate_from_current_filter_;
+    // Tracks if the stream end has been sent with an encode/decodeData call, to handle the case
+    // where commonContinue is called with the only information being a terminal fin.
+    bool end_stream_sent_with_data_;
```

Comment:
Can we add this bool to the bitfield below? IIRC the reason the one above isn't in it is because it needs to be passed by address somewhere.